File: Programação I/URI/Salario.java
import java.util.Scanner;
public class Salario {
    public static void main( String args[]) {
        Scanner input = new Scanner( System.in);

        int numero ;
        float valor ;
        float totals ;
        float horas ;

        System.out.println("Digite o numero: ");
        numero = input.nextInt();
        System.out.println("Digite o valor por hora: ");
        valor = input.nextInt();
        System.out.println("Digite a quantidade de horas trabalhadas: ");
        horas = input.nextInt();

        totals = valor * horas ;


        System.out.println("NUMBER = " + numero);
        System.out.printf("SALARY = %.2f\n", totals);
    }
}

File: Estrutura de Dados/Prática/Salario.c
#include <stdio.h>
int main(){
    int number;
    double horas, salario, total;
    scanf("%i %lf %lf", &number, &horas, &salario);
    total = horas * salario;
    printf("NUMBER = %i\nSALARY = U$ %.2lf\n", number, total);
    return 0;
}